Keypad/Keypad.js

```diff
@@ -100,7 +135,22 @@ const KeypadBase = kind({
 	name: 'Key',
 
 	propTypes: {
+		/**
+		 * Applies a disabled style and the control becomes non-interactive.
+		 *
+		 * @type {Boolean}
+		 * @default false
```

Comment:
There is no `defaultProps` in `Key`'s `disabled`. So I'm curious why the default value is false. If it is defined in another component, then please omit it here.